### ISSUE TITLE
Quick save slot not shown in saves list

### ISSUE THREAD

**Issue by abelsromero** [Try/OpenGothic#543]
Quick save is saved in slot (`save_slot_0.sav`) but it does not appear in the Save/Load menu. You can still load with F9, but not from the menu.

**Comment by Try** [comment]
Hi, @abelsromero !

Similar to original game you need to enable it in `Gothic.ini` file:
```
[GAME]
useQuickSaveKeys=1
```

Made it `1`, by default in ae841fbf

**Comment by abelsromero** [comment]
That helps make it more consistent, but still in the original there was a "- Quick Save -" option added at the end of the list (above the Back) that does not show in OpenGothic.

![image](https://github.com/Try/OpenGothic/assets/5781153/65341588-e696-4d40-a3e0-3d09203b08e6)

PS: I am reporting anything I see in hopes of helping, the work is amazing, keep it up. No hurry, I wish I had time to learn cpp to help :sweat_smile:

**Comment by abelsromero** [comment]
One thing I noticed is the original does not allow quick-loading when the menu is displayed. I think the same could be applied here, this fixes some quicks I am seeing like pressing F9 on the saves menu still shown after loading.

**Comment by Try** [comment]
> there was a "- Quick Save -" option

Done at [5a449c0](https://github.com/Try/OpenGothic/commit/5a449c0e085404c24e0dbddf102110444ba8f071)

> does not allow quick-loading when the menu is displayed

Done at [19a8400](https://github.com/Try/OpenGothic/commit/19a8400b706eaa791b2d192b15bfa34a43497065)

**Comment by abelsromero** [comment]
> > there was a "- Quick Save -" option
> 
> Done at [5a449c0](https://github.com/Try/OpenGothic/commit/5a449c0e085404c24e0dbddf102110444ba8f071)

I am sorry to say that this does not seem to be working. I rebuild including the new commit and I don't see any change, I was expecting something like this 
![image](https://github.com/Try/OpenGothic/assets/5781153/a2841cad-4687-457b-891a-3e37f963205f)

**Comment by Try** [comment]
Here what I do have on steam version:
![изображение](https://github.com/Try/OpenGothic/assets/122894/a5a3ffec-af2e-4643-af18-0a78d13c025b)

how menu look a like for you?

**Comment by abelsromero** [comment]
Thanks, I simply don't see it and I can still use F9 to load
 
![image](https://github.com/Try/OpenGothic/assets/5781153/5e0be6de-2380-48f2-adad-7b5fbdae8e12)

I did a clean build just in case (completely deleted the `build` folder and run cmake setup and build again) and my git history shows I have the changes, weird...
![image](https://github.com/Try/OpenGothic/assets/5781153/2482406e-a538-4c89-a18a-619735528282)

**Comment by Try** [comment]
Thank for additional info.

> I simply don't see it and I can still use F9 to load

Can you clarify: is it purely display issue? Can you navigate, with keyboard, to presumed quick-save line and load it?

**Comment by abelsromero** [comment]
> Can you clarify: is it purely display issue? 

I'd say yes.

>Can you navigate, with keyboard, to presumed quick-save line and load it?

Can't navigate, the option does not appear, shown like in the comment above https://github.com/Try/OpenGothic/issues/543#issuecomment-1863986504.

**Comment by Try** [comment]
> Can't navigate, the option does not appear,

Then the UI element must be hidden, can you check with config variable in Gothic.ini set:
```
[GAME]
useQuickSaveKeys=1
```
?

**Comment by abelsromero** [comment]
> [GAME]
> useQuickSaveKeys=1

With that added the option is shown, but then (sorry to nitpick):
1. It's not enabled by default as https://github.com/Try/OpenGothic/commit/ae841fbf0aa8476a1aef0cc925053a59e004682e implies. I deleted Gothic.ini and it was re-created without it.
2. The original game also requires the option to use F5 and F9. If the aim is to reproduce the original as-is the keyboard shortcuts should also be affected by the ini right?

**Comment by Try** [comment]
> It's not enabled by default as https://github.com/Try/OpenGothic/commit/ae841fbf0aa8476a1aef0cc925053a59e004682e implies. I deleted Gothic.ini and it was re-created without it.

Probably there is a confusion on what is "defaults". In OpenGothic variable resolution works as follows:
1. Gothic.ini in Gothic2Notr.exe directory
2. Gothic.ini in vanilla/system
3. the 'default' values

Apparently in your case there is already some data in `2.`

> The original game also requires the option to use F5 and F9.

Done

**Comment by abelsromero** [comment]
> Apparently in your case there is already some data in `2.`

TLDR; YES.

I did some testing, in case someone elses stumbles upon this at some point:
* Clean installation and running the game at least once.
  * Steam contains `system/Gothic.ini` with `useQuickSaveKeys=0`.
  * GOG does NOT contain `system/Gothic.ini`
  * None of them contain `Gothic2Notr.exe`

For my tests, I am using a clean install copy of Steam. I copied the files to a different folder without doing any run. When I run a comparison with Steam, I should not be affected by modifications or runs on Steam.

Based on that and now with this fix https://github.com/Try/OpenGothic/commit/6b99ead5d9ed87a14ecd10908ea83ed70a45ac01, I can confirm all is working as expected now.
Also tested what happens when all .ini are deleted, in that case QuickSave are enabled but there's no `useQuickSaveKeys` anywhere.
